### DISCUSSION TITLE
Drons Alerts

### DISCUSSION THREAD

**Discussion by vadimvorenas** [notnotmelon/factorissimo-2-notnotmelon#77] [Q&A] [answered]
Please add button "disabled auto drons added". Every time on boot i have only me factories alert "no added drons". It's a big hindrance 
![Factorio Screenshot 2024 11 03 - 01 35 34 74](https://github.com/user-attachments/assets/7c3ef2cc-e2f0-4c10-88fd-68c00318acef)

**Comment by notnotmelon** [comment]
Thanks for the post. I believe its the same as here: https://github.com/notnotmelon/factorissimo-2-notnotmelon/issues/79

**Reply by vadimvorenas** [reply]
Thx for answer, but on version 3.5.12 this problem remains. Better possible - added capasity "Disabled this functional", i can add drons myself

**Reply by vadimvorenas** [reply]
Even when station has robots, i receive notifications (when load save game)
![Factorio Screenshot 2024 11 03 - 17 58 18 34](https://github.com/user-attachments/assets/66749b8a-2b6c-41f6-99a1-875c6800b35d)

**Reply by vadimvorenas** [reply]
It happened even i changed me settings in different mods on played game.

**Reply by vadimvorenas** [reply]
Now i change it, for now it help me
in roboport.lua

```
    factory.roboport_upgrade = {
        roboport = roboport,
        storage = storage,
        requester = requester,
        -- inital_ten_robot_request = inital_ten_robot_request,
        num_robots_requested = 0,
        item_request_proxies = (factory.roboport_upgrade and factory.roboport_upgrade.item_request_proxies) or {}
    }
```

**Reply by vadimvorenas** [reply]
If i know how it to add in menu, i share be, try makes it

**Reply by MaxAstro** [reply]
Came here with this same problem. Didn't expect researching a technology to spam me with 60+ alerts across my entire base...

Worse, I am using a mod that replaces the vanilla construction bots, so I can't get the alerts to go away no matter what I do.

**Reply by MaxAstro** [reply]
Note for anyone having the same problem as me:

The fix vadimvorenas posted didn't work for me, so instead I resorted to forcing set_default_roboport_construction_robot_request to return 0. That worked.

**Reply by vadimvorenas** [reply]
> Note for anyone having the same problem as me:
> 
> The fix vadimvorenas posted didn't work for me, so instead I resorted to forcing set_default_roboport_construction_robot_request to return 0. That worked.

Yes, me fix didn't clear current notification, but if you removed all problem manual - new notification doesn't add

**Reply by MaxAstro** [reply]
For me the notifications kept coming back after changing any startup settings.

**Reply by vadimvorenas** [reply]
any setting for the mod, or for different mods?

**Comment by Black0721** [comment]
In my case, disabling Robot World Continued stopped the error messages. However, when I re-enabled Robot World Continued and performed construction activities, the same drone-related errors began accumulating again.

**Reply by vadimvorenas** [reply, marked as answer]
In a last version mod, no more Robots. And I dont have errors messages now.
